### PR TITLE
Added en-gb-locale

### PR DESCRIPTION
**Pull request by tcrands**
Added a new locale for en-gb that supports the time and date formats.

### CODE REVIEW

File: src/locale/en-gb.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
+import dayjs from 'dayjs'
+
+const locale = {
+  name: 'en',
```
Comment (iamkun) [inline src/locale/en-gb.js]: en-gb

**Comment by iamkun** [comment]
What's the difference between en-gb and built in en locale, please?

**Comment by tcrands** [comment]
The difference is the time and data formats. It says the built in en locale is US so this won't work for the UK.

Alternatively we could make the default en locale UK based?

**Comment by iamkun** [comment]
Is `formats` the only different part?

**Comment by tcrands** [comment]
Yes I believe formats is the only difference

File: src/locale/en-gb.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    const b = number % 10
+    let output = 'th'
+    if (b === 1) {
+      output = 'st'
+    } else if (b === 2) {
+      output = 'nd'
+    } else if (b === 3) {
+      output = 'rd'
+    }
+    return number + output
```
Comment (iamkun) [inline src/locale/en-gb.js]: ordinal = (number) => {
    const s = ['th', 'st', 'nd', 'rd']
    const v = number % 100
    return `[${number}${(s[(v - 20) % 10] || s[v] || s[0])}]`
  }

This looks better

**Comment by iamkun** [comment]
npm run lint first, pls

**Pull request merged**